internal/file.go

```diff
@@ -187,11 +187,11 @@ func (fh *FileHandle) partSize() uint64 {
 	var size uint64
 
 	if fh.lastPartId < 1000 {
-		size = 5 * 1024 * 1024
+		size = fh.inode.fs.flags.MPUPartSize
```

Comment:
This behavior may confuse users.  Instead, how about using a fixed size when provided by the flag and this gradually increasing size when not provided?

Reply:
OK, we can further discussing this after we accept this proposal